Core/clim-basic/extended-streams/pointer-tracking.lisp

```diff
@@ -191,6 +191,7 @@
               ;; the other hand, we pass events for TRACKED-SHEET (or
               ;; all events if MULTIPLE-WINDOW is true) to TRACK-EVENT.
               do (cond ((not (or multiple-window
+                                 (and (typep event 'keyboard-event) (keyboard-handler state))
                                  (eql tracked-sheet (event-sheet event))))
```

Comment:
I think that a solution that fits better how the tracking-pointer is implemented would invovle:
- extending with-pointer-grabbed to accept `:keyboard t` (that is, change port-grab-pointer - for clx that would be adding the event to the event mask)
- wrapping the body in `with-input-focus` on a sheet

Reply:
I think taht  the second suggestion (`with-input-focus`) is better: in principle to track the pointer on a single window grabbing the pointer isn't necessary . 
Anyway I think that that `port-grab-pointer` needs a review (see #1120 and #899)